packages/components/src/utils/animate.ts

```diff
@@ -61,13 +61,13 @@ export const KEYFRAMES = {
       offset: 0,
       ...keyframeDefaults,
       opacity: 0,
-      transform: 'translateY(-3rem)',
+      top: '-3rem',
```

Comment:
I would probably add a comment here explaning why were not using `transform` with a link to that piece of mdn regarding `fixed`? — I can imagine somebody legitimately changing this back in the name of performance 🤔

Reply:
makes sense, i added a comment there